### ISSUE TITLE
Update the SRS document

### ISSUE THREAD

**Issue by Dominique-Da-Silva** [COS301-SE-2024/MyCity#12]
Update the following key features in the SRS document:
FRs should be more granular.
NFR
- localization should be moved to accessibility.
- integration to extensibility.
- offline should be a core requirement.
- platform independence to extensibility.
- Usability should be added to the non functional requirements.

Subsystems should be linked to FR and not NFR.
